src/PrestaShopBundle/Entity/Employee/Employee.php

```diff
@@ -0,0 +1,699 @@
+     */
+    private Collection $sessions;
+
+    /**
+     * @ORM\Column(name="firstname", type="string")
+     */
+    private string $firstName;
+
+    /**
+     * @ORM\Column(name="lastname", type="string")
+     */
+    private string $lastName;
+
+    /**
+     * @ORM\Column(name="email", type="string")
+     */
+    private string $email;
+
+    /**
+     * @ORM\Column(name="passwd", type="string")
+     */
+    private string $password;
+
+    /**
+     * @ORM\Column(name="last_passwd_gen", type="datetime", options={"default": "CURRENT_TIMESTAMP"})
+     */
+    private DateTime $passwordLastGeneration;
+
+    /**
+     * @ORM\Column(name="reset_password_token", type="string", length=40, nullable=true)
+     */
+    private ?string $resetPasswordToken;
+
+    /**
+     * @ORM\Column(name="reset_password_validity", type="datetime", nullable=true)
+     */
+    private ?DateTime $resetPasswordValidity;
+
+    /**
+     * @ORM\Column(name="default_tab", type="integer", options={"default": 0, "unsigned": true})
+     */
+    private int $defaultTabId;
+
+    /**
+     * @ORM\Column(name="active", type="boolean", options={"default": 0})
+     */
+    private bool $active;
+
+    /**
+     * @ORM\Column(name="last_connection_date", type="date", nullable=true)
+     */
+    private ?DateTime $lastConnectionDate;
+
+    /**
+     * @ORM\Column(name="has_enabled_gravatar", type="boolean", options={"default": 0})
+     */
+    private bool $hasEnabledGravatar;
+
+    /**
+     * @ORM\Column(name="stats_date_from", type="date", nullable=true)
+     */
+    private ?DateTime $statsDateFrom;
+
+    /**
+     * @ORM\Column(name="stats_date_to", type="date", nullable=true)
+     */
+    private ?DateTime $statsDateTo;
+
+    /**
+     * @ORM\Column(name="stats_compare_from", type="date", nullable=true)
+     */
+    private ?DateTime $statsCompareFrom;
+
+    /**
+     * @ORM\Column(name="stats_compare_to", type="date", nullable=true)
+     */
+    private ?DateTime $statsCompareTo;
+
+    /**
+     * @ORM\Column(name="stats_compare_option", type="integer", options={"default": 1, "unsigned": true})
+     */
+    private int $statsCompareOption;
+
+    /**
+     * @ORM\Column(name="preselect_date_range", type="string", length=32, nullable=true)
+     */
+    private ?string $preselectDateRange;
+
+    /**
+     * @ORM\Column(name="bo_color", type="string", length=32, nullable=true)
+     */
+    private ?string $boColor;
+
+    /**
+     * @ORM\Column(name="bo_theme", type="string", length=32, nullable=true)
+     */
+    private ?string $boTheme;
+
+    /**
+     * @ORM\Column(name="bo_css", type="string", length=64, nullable=true)
+     */
+    private ?string $boCss;
+
+    /**
+     * @ORM\Column(name="bo_width", type="integer", options={"default": 0, "unsigned": true})
+     */
+    private int $boWidth;
+
+    /**
+     * @ORM\Column(name="bo_menu", type="boolean", options={"default": 1})
+     */
+    private bool $boMenu;
```

Comment:
When I see that I tell myself that it could go to another entity

Reply:
This is no longer used, maybe we can use this opportunity to remove this 💩?

bo_color
bo_width
bo_theme (there is only default)
bo_menu

Ping @kpodemski correct me if I miss something

Reply:
I don't mind, for now I had to leave them in the Doctrine entity or the doctrine command would automatically delete these fields, so I had to sync the content of the entity with the db structure table

We could clean those, but in another PR then